.prettierrc

```diff
@@ -1,27 +0,0 @@
-{
```

Comment:
lets keep this file for js pretify

we want to pretify all js files.
such that prettier runs by default.